rune/libenclave/epm/procmaps.go

```diff
@@ -6,7 +6,8 @@ import (
 )
 
 const (
-	EnclavePath = "/dev/sgx/enclave"
+	EnclavePath  = "/dev/sgx/enclave"
+	EnclavePath1 = "/sgx/enclave"
```

Comment:
In which case we will see such a path?

Reply:
If an enclave is stored into pool, the pathname will be /sgx/enclave. If the enclave from pool is stored into pool again, the pathname /sgx/enclave will be used.

Reply:
The control flow is:
1. container A inits enclave and stored it into enclave pool in rune side. In epm side, the enclave fd's pathname will be /sgx/enclave
2. start another container B, it will obtain the enclave from pool, the enclave's fd pathname is /sgx/enclave, so once realesed, it will be stored into pool by pathname /sgx/enclave (retrieve enclave fd and maps info by the pathname in rune side)